src/content/docs/id/configuring/hyprland.md

```
@@ -0,0 +1,110 @@
+---
+title: Hyprland
+description: Konfigurasi Hyprland
+sidebar:
+  order: 3
+---
+
+<link rel="stylesheet" href="/src/styles/tables.css">
+
+# Struktur Konfigurasi
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)